### PR TITLE
Event batch with changes made

### PR DESCRIPTION
**Pull request by nataliyaned**
I made the changes and uploaded via the same process as before.

Adding quotation marks around the links required changes to dozens of events throughout the file, so this seemed to be the simplest way. 

Let me know if there's a preferred way of doing things. I'm still learning so the feedback definitely helps.

### CODE REVIEW

File: actions.csv [hunk 1]
```diff
@@ -1,142 +1,160 @@
...
-103,2019/05/11,"https://www.fastcompany.com/90348304/exclusive-tech-workers-organize-protest-against-palantir-on-the-github-coding-platform, https://gist.github.com/noahzgordon/29af8c13106ddc591ccaa1ea0ce6af74",protest,ethics,"Tech workers organized by the Tech Workers Coalition are protesting the Palantir's contract with ICE on the company's GitHub projects. The plan calls for organized waves of digital protesters to post a prepared message to their GitHub boards. The protest appeals to workers to consider their role, even if indirectly, in Palantir’s work with ICE. It also encourages people to add their own personal sentiments.",usa,palantir,None,"ice, industry_solidarity, twc",organizejs
-104,2019/05/22,"https://www.vox.com/recode/2019/5/22/18635604/amazon-shareholder-meeting-2019-climate-change-proposal, https://gizmodo.com/jeff-bezos-wouldn-t-even-come-on-stage-to-listen-to-his-1834957220",protest,ethics,"More than 50 Amazon employees from Amazon Employees for Climate Justice attended the shareholder meeting, where Emily Cunningham delivered an impassioned speech to shareholders urging them to support the resolution for Amazon to adopt a strong climate plan. Jeff Be
```
Comment (organizejs) [inline actions.csv]: unfortunately the continuous integration / validation code will freakout with this kind of syntax, but im making a code update that will make it more flexible. 

Why triple quotation?

Comment (organizejs) [inline actions.csv]: unfortunately the validation pipeline requires a url this field, so `Page 143 of Ghost Work ...` will cause it to fail. Can we add that reference in the description area instead?

Sorry!

**Comment by organizejs** [comment]
addresses: #170 #167 #166 #115 #88

File: actions.csv (hunk 1 above)
Comment (nataliyaned) [reply actions.csv]: Ahhh my mistake. I was making edits through Excel and it was showing up as single quotations. Fixed in the latest version.

Comment (nataliyaned) [reply actions.csv]: Got it

**Pull request closed without merging**